byu-oit-appdev/hr-pro-sync
# Fix one file

* Issue-4_Link-new-code (#5)

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Test body parms

* Test body parms

* Test body parms

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Verify JWT from WSO2

* Refactor for testing (#7)

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Add event handler to lambda function.

* Test body parms

* Test body parms

* Test body parms

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Add S3 bucket lookup

* Verify JWT from WSO2

* Filter by event types

* Filter by event types

* Filter by event types

* Filter by event types

* Filter by event types

* Filter by event types

* Filter by event types

* Filter by event types

* Filter by event types

* Cleanup code

* Cleanup code

* check for null emails

* check for null emails

* Delete email when there are no HOME or BUSN emails.

* Fix syntax error.

* Fix syntax error.

* Fix syntax error.

* Fix syntax error.

* Fix syntax error.

* Fix syntax error.

* Fix syntax error.

* Refactor for testing

* Refactor for testing

* Refactor for testing

* Refactor for testing

* Refactor for testing

* Refactor for testing

* Refactor for testing

* Cleanup console messages

* Cleanup console messages

* Cleanup console messages
